content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,145 +124,114 @@ enum Level {
      main = "src/main.ts"
      compatibility_date = "2022-11-07"
 
    + [vars]
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
+   + DATABASE_URL = "prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
+   ```
+
+3. Install the Accelerate Client extension
+
+   ```bash
+   npm install @prisma/extension-accelerate
    ```
 
 You are now ready to generate a Prisma Client.
 
-## 8. Generate a Prisma Client
+## 6. Generate a Prisma Client
 
-Next, you'll generate a Prisma Client that connects through the [Data Proxy](/data-platform/classic-projects/data-proxy) over HTTP.
+Next, you'll generate Prisma Client that connects to your database through [Prisma Accelerate](/data-platform/accelerate) over HTTP.
 
 ```terminal
 npx prisma generate --no-engine
 ```
 
 <Admonition>
 
-Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--data-proxy` flag.
+Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--accelerate` flag.
 
 ```terminal
-npx prisma generate --data-proxy
+npx prisma generate --accelerate
 ```
 
 </Admonition>
 
 The generated Client has a smaller bundle size and is optimized for edge environments like Cloudflare Workers.
 
-The smaller bundle size is due to the fact that the interfaces talking to the database (the [Prisma engines](/concepts/components/prisma-engines)) are no longer bundled with Prisma Client as this logic is now handled by the Data Proxy.
+The smaller bundle size is due to the fact that the interfaces talking to the database (the [Prisma engines](/concepts/components/prisma-engines)) are no longer bundled with Prisma Client as this logic is now handled by Prisma Accelerate.
 
-## 9. Develop the Cloudflare Worker function
+## 7. Develop the Cloudflare Worker function
 
 You're now ready to create a Cloudflare Worker. Create a `src/index.ts` file with the following code:
 
 ```ts
 import { PrismaClient } from '@prisma/client/edge'
-const prisma = new PrismaClient()
-
-addEventListener('fetch', (event) => {
-  event.respondWith(handleEvent(event))
-})
-
-async function handleEvent(event: FetchEvent): Promise<Response> {
-  const { request } = event
-
-  // waitUntil method is used for sending logs, after response is sent
-  event.waitUntil(
-    prisma.log
-      .create({
-        data: {
-          level: 'Info',
-          message: `${request.method} ${request.url}`,
-          meta: {
-            headers: JSON.stringify(request.headers),
-          },
-        },
+import { withAccelerate } from '@prisma/extension-accelerate'
+
+export interface Env {
+  DATABASE_URL: string
+}
+
+export default {
+  async fetch(
+    request: Request,
+    env: Env,
+    ctx: ExecutionContext
+  ): Promise<Response> {
+    const prisma = new PrismaClient({
+      datasourceUrl: env.DATABASE_URL,
+    }).$extends(withAccelerate())
+
+    const { data, info } = await prisma.logs
+      .findMany({
```

Comment:
Where do we actually write the logs?
Otherwise this will just be querying an empty table?

Reply:
@janpio I'll add a simple create query so that we're actually writing the logs.